cf/commands/servicekey/delete_service_key.go

```diff
@@ -86,7 +86,7 @@ func (cmd DeleteServiceKey) Run(c *cli.Context) {
 	}
 
 	serviceKey, err := cmd.serviceKeyRepo.GetServiceKey(serviceInstance.Guid, serviceKeyName)
-	if err != nil {
+	if err != nil || serviceKey.Fields.Guid == "" {
```

Comment:
Shouldn't we check first specifically for `serviceKey.Fields.Guid == ""` since that should always produce an error? Since if the key exists the GUID cannot be `""`.  Correct?